frontend/templates/scale-modal/scale-editor.tmpl

```diff
@@ -0,0 +1,22 @@
+{{#unless isNew}}
```

Comment:
The delete button for scales is never shown, probably because this is always true?

Reply:
This will be addressed in #568 (once I resurrect it and port it to the appropriate branch).